Update constants.ts

baishakhi9931 opened the pull request:
[ISSUE](https://github.com/erxes/erxes/issues/ISSUE)

### Context

Your context here.  Additionally, any screenshots.  Delete this line.


// Delete the below section once completed
### PR Checklist
- [ ] Description is clearly stated under Context section
- [ ] Screenshots and the additional verifications are attached

baishakhi9931 commented:
I have updated with adding Standard Arabic language because it is one of the most common languages used in the world including all mentioned here.

Maskgirl commented:
> I have updated with adding Standard Arabic language because it is one of the most common languages used in the world including all mentioned here.

did u find and changed some docker-compose.yml file?

baishakhi9931 commented:
> > I have updated with adding Standard Arabic language because it is one of the most common languages used in the world including all mentioned here.
> 
> did u find and changed some docker-compose.yml file?

No, I have only added the Arabic language that's it.

lokeshrajgadkar reviewed (approved):
Your changes seems fine.

Maskgirl commented:
> Your changes seems fine.

https://github.com/erxes/erxes/pull/2457 check my pr request once

baishakhi9931 commented:
Kindly check it @erxes

batamar commented:
@baishakhi9931 Hi. You also need to provide corresponding sd_ar.json file
